lms/djangoapps/learner_home/serializers.py

```diff
@@ -379,6 +390,16 @@ def get_entitlements(self, instance):
         else:
             return {}
 
+    def get_programs(self, instance):
+        """
+        If this enrollment is part of a program, include information about the program and related programs
+        """
+        programs = self.context['programs'].get(str(instance.course_id))
+        if programs:
+            return ProgramsSerializer({"relatedPrograms": programs}, context=self.context).data
```

Comment:
I am not sure tbh, I agree that this feel unnecessary but I feel like changing this would require even more work from the frontend. It is ugly but it isn't terrible.

Reply:
Had this discussion w/ @muselesscreator, the idea was that we might have other programs-related info to include in the future so this was an intentional nesting choice.